docs/config.html

```diff
@@ -484,6 +485,65 @@ <h2 id="build-env" class="title-2">[BuildEnv]</h2>
   </p>
 </section>
 
+<section class="mt4">
+  <h2 id="sandbox" class="title-2">[Sandbox]</h2>
+  <ul class="bulleted-list">
+    <li>
+      <div>
+        <h3 class="mt1 f6 lh-title">
+          Tool
+        </h3>
+
+        <p>
+          The binary to use for sandboxing build/test actions. Typically this
+          is <code>please_sandbox</code>.<br />
+          It receives the program to execute and all its arguments in its <code>argv</code>,
+          it is expected to do whatever it needs to then <code>exec</code> the real action.
```

Comment:
Worth mentioning that `please_sandbox` only works on linux?

Reply:
yeah, fair